client/setdownedstate.lua

```diff
@@ -0,0 +1,150 @@
+local deadAnimDict = "dead"
+local deadAnim = "dead_a"
+
+local function drawTxt(x, y, width, height, scale, text, r, g, b, a, _)
+    SetTextFont(4)
+    SetTextProportional(false)
+    SetTextScale(scale, scale)
+    SetTextColour(r, g, b, a)
+    SetTextDropShadow(0, 0, 0, 0, 255)
+    SetTextEdge(2, 0, 0, 0, 255)
+    SetTextDropShadow()
+    SetTextOutline()
+    SetTextEntry("STRING")
+    AddTextComponentString(text)
+    DrawText(x - width / 2, y - height / 2 + 0.005)
+end
+
+local function isEmsOnDuty()
+    QBCore.Functions.TriggerCallback('hospital:GetDoctors', function(medics)
+        return medics > 0
+    end)
+end
```

Comment:
This won't actually return in the isEmsOnDuty function, but rather just end the execution of the callback, so this is the fix
```suggestion
local function isEmsOnDuty()
    local p = promise.new()
    QBCore.Functions.TriggerCallback('hospital:GetDoctors', function(medics)
        p:resolve(medics > 0)
    end)

    return Citizen.Await(p)
end
```